doc/user/ANIMATIONS.md

```diff
@@ -0,0 +1,40 @@
+# Animations
+
+F3D is able to play animations for any files which contain them.
+Play them either interactively or by selecting a specific time to display.
+For files containing multiple animations, F3D allows the user to either play each animation separately or play all animations at the same time.
+## Demonstration
+This specific example uses an animation file which can be downloaded [here](https://github.com/f3d-app/f3d/blob/606089959c9520085a9cbf70660fb0ffc68fb934/testing/data/InterpolationTest.glb).
+
+<img width="1024" alt="1" src="https://github.com/f3d-app/f3d/assets/2395780/1402c513-b3fd-454b-84a4-738eb41a1ccf">
```

Comment:
Where are these images coming from?

Reply:
no idea, here is the right version https://github.com/f3d-app/f3d-media/pull/1